ouf.lua

```diff
@@ -564,6 +564,97 @@ function oUF:Spawn(unit, overrideName)
 	return object
 end
 
+function oUF:SpawnNamePlates(namePrefix, nameplateCallback, nameplateCVars)
+	argcheck(nameplateCallback, 3, 'function', 'nil')
+	argcheck(nameplateCVars, 4, 'table', 'nil')
+	if(not style) then return error('Unable to create frame. No styles have been registered.') end
```

Comment:
Ugh, there's actually more of them around.
Leave it and we'll correct it with a different PR post-merge.